DEVELOPMENT.md

```diff
@@ -44,6 +48,7 @@ sed -e 's/^/    /' ${HOME}/.kube/config | cat contrib/examples/cluster.yaml - |
 ```
 
 # Using `kcp` as a library
+
```

Comment:
If I am not mistaken,  `DefaultConfig` is no more. So the code below must look completely differently now, using option structs.